src/user-actions.ts

```diff
@@ -0,0 +1,45 @@
+export default interface User {
+  id: string;
+  login: string;
+  password: string;
+  age: number;
+  isDeleted: boolean;
+}
+
+interface UserRes {
+  id: string;
+  login: string;
+  age: number;
+}
+
+export function getResUser(user: User): UserRes {
```

Comment:
the same https://github.com/alexandrali/node_ht2/pull/1/files#r1068368587

Reply:
fixed